templates/charon_plugin.conf.erb

```diff
@@ -0,0 +1,25 @@
+#
```

Comment:
can you please convert this to an epp template? We require epp for all new templates.

Reply:
changed template to epp